### ISSUE TITLE
Soul Forge server crash

### ISSUE THREAD

**Issue by Sir-Will** [WayofTime/BloodMagic#1177]
BloodMagic-1.10.2-2.1.11-80
````
Time: 7/23/17 10:33 AM
Description: Ticking block entity

java.lang.IllegalArgumentException: Invalid UUID string: 
	at java.util.UUID.fromString(UUID.java:194)
	at WayofTime.bloodmagic.api.util.helper.NetworkHelper.getSoulNetwork(NetworkHelper.java:45)
	at WayofTime.bloodmagic.item.ItemLavaCrystal.getContainerItem(ItemLavaCrystal.java:34)
	at WayofTime.bloodmagic.tile.TileSoulForge.consumeInventory(TileSoulForge.java:250)
	at WayofTime.bloodmagic.tile.TileSoulForge.craftItem(TileSoulForge.java:173)
	at WayofTime.bloodmagic.tile.TileSoulForge.func_73660_a(TileSoulForge.java:115)
	at org.spongepowered.common.event.tracking.TrackingUtil.tickTileEntity(TrackingUtil.java:231)
	at net.minecraft.world.WorldServer.updateTileEntity(WorldServer.java:2831)
	at net.minecraft.world.WorldServer.redirect$onUpdateTileEntities$zkb000(WorldServer.java:2818)
	at net.minecraft.world.World.func_72939_s(World.java:6049)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:2185)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:709)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471)
	at java.lang.Thread.run(Thread.java:748)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at java.util.UUID.fromString(UUID.java:194)
	at WayofTime.bloodmagic.api.util.helper.NetworkHelper.getSoulNetwork(NetworkHelper.java:45)
	at WayofTime.bloodmagic.item.ItemLavaCrystal.getContainerItem(ItemLavaCrystal.java:34)
	at WayofTime.bloodmagic.tile.TileSoulForge.consumeInventory(TileSoulForge.java:250)
	at WayofTime.bloodmagic.tile.TileSoulForge.craftItem(TileSoulForge.java:173)
	at WayofTime.bloodmagic.tile.TileSoulForge.func_73660_a(TileSoulForge.java:115)
	at org.spongepowered.common.event.tracking.TrackingUtil.tickTileEntity(TrackingUtil.java:231)
	at net.minecraft.world.WorldServer.updateTileEntity(WorldServer.java:2831)
	at net.minecraft.world.WorldServer.redirect$onUpdateTileEntities$zkb000(WorldServer.java:2818)

-- Block entity being ticked --
Details:
	Name: BloodMagic:TileSoulForge // WayofTime.bloodmagic.tile.TileSoulForge
	Block type: ID #913 (tile.BloodMagic.soulForge // WayofTime.bloodmagic.block.BlockSoulForge)
	Block data value: 0 / 0x0 / 0b0000
	Block location: World: (-3485,80,2457), Chunk: (at 3,5,9 in -218,153; contains blocks -3488,0,2448 to -3473,255,2463), Region: (-7,4; contains chunks -224,128 to -193,159, blocks -3584,0,2048 to -3073,255,2559)
	Actual block type: ID #913 (tile.BloodMagic.soulForge // WayofTime.bloodmagic.block.BlockSoulForge)
	Actual block data value: 0 / 0x0 / 0b0000
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:6049)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:2185)

-- Affected level --
Details:
	Level name: DIM-5
	All players: 1 total; [EntityPlayerMP['braun_the_pirate'/1539, l='DIM-5', x=-3485.48, y=80.00, z=2455.54]]
	Chunk stats: ServerChunkCache: 81 Drop: 0
	Level seed: -8913466909937300889
	Level generator: ID 00 - default, ver 1. Features enabled: true
	Level generator options: 
	Level spawn location: World: (0,64,0), Chunk: (at 0,4,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 37900217 game time, 37900217 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 90751 (now: false), thunder time: 129458 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false

**Comment by Arcaratus** [comment]
Is Blood Arsenal installed and if so are you crafting the Blood Diamond?

**Comment by Sir-Will** [comment]
yes

**Comment by Arcaratus** [comment]
If you're using Minecraft 1.11.2 then you can update Blood Arsenal (which
contains a new recipe for the Blood Diamond) that doesn't crash.

This is still a problem to be fixed on the Blood Magic side though.
 
EDIT: There is now a new version of Blood Arsenal for 1.10.2 which contains the new recipe.

_Sent from my Samsung SM-G930T using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by TehNut** [comment]
Should be fixed in 1.12 by 441e7b8952e4f84bb0a99814904cc3ef55583c66
